Sources/SecureXPC/Server/ClientRequirement.swift

```diff
@@ -163,6 +170,34 @@ fileprivate protocol MessageAcceptor {
     func isEqual(to acceptor: MessageAcceptor) -> Bool
 }
 
+fileprivate struct HardenedMessageAcceptor: MessageAcceptor {
+	func shouldAcceptMessage(connection: xpc_connection_t, message: xpc_object_t) -> Bool {
+		guard let code = SecCodeCreateWithXPCConnection(connection, andMessage: message) else {
+			return false
+		}
+
+		var staticCode: SecStaticCode?
+		var codeSignInformation: CFDictionary?
+
+		guard SecCodeCopyStaticCode(code, [], &staticCode) == errSecSuccess,
+			  let staticCode = staticCode,
+			  SecCodeCopySigningInformation(staticCode, [], &codeSignInformation) == errSecSuccess,
+			  let codeSignInformation = codeSignInformation as NSDictionary?,
+			  let codeFlagsInt = codeSignInformation[kSecCodeInfoFlags] as? NSNumber
```

Comment:
Can this directly be cast to `UInt32` here and then later not need to use `codeFlagsInt.uint32Value`?